api/rhsm-subscriptions-api-spec.yaml

```diff
@@ -586,126 +586,6 @@ paths:
           $ref: "../spec/error-responses.yaml#/$defs/InternalServerError"
       tags:
         - capacity
-  /hosts/products/{product_id}:
-    description: 'Operations for hosts for a given account and product'
-    parameters:
-      - name: product_id
-        in: path
-        required: true
-        schema:
-          $ref: "#/components/schemas/ProductId"
-        description: "The ID for the product we wish to query"
-      - name: offset
-        in: query
-        schema:
-          type: integer
-        description: "The number of items to skip before starting to collect the result set"
-      - name: limit
-        in: query
-        schema:
-          type: integer
-          minimum: 1
-          maximum: 100
-        description: "The numbers of items to return"
-    get:
-      deprecated: true
-      summary: "Fetch hosts matching report criteria."
-      operationId: getHosts
-      parameters:
-        - name: sla
-          in: query
-          schema:
-            $ref: '#/components/schemas/ServiceLevelType'
-          description: "Include only hosts for the specified service level."
-        - name: usage
-          in: query
-          schema:
-            $ref: '#/components/schemas/UsageType'
-          description: "Include only hosts for the specified usage level."
-        - name: uom
-          in: query
-          schema:
-            $ref: '#/components/schemas/Uom'
-          description: "Filter hosts to those that contribute to a specific unit of measure"
-        - name: display_name_contains
-          description: Include only hosts containing the specified display name. Passing an empty string behaves the same way as passing null
-          schema:
-            type: string
-            #Default to empty string to prevent complex jpql statement later
-            default: ''
-          in: query
-        - name: beginning
-          in: query
-          required: false
-          schema:
-            type: string
-            format: date-time
-          description: "Defines the start of the report period. Dates should be provided in ISO 8601
-            format but the only accepted offset is UTC. E.g. 2017-07-21T17:32:28Z.
-            Only applicable for OpenShift-dedicated-metrics and OpenShift-metrics products"
-        - name: ending
-          in: query
-          required: false
-          schema:
-            type: string
-            format: date-time
-          description: "Defines the end of the report period. Dates should be provided in ISO 8601
-            format but the only accepted offset is UTC. E.g. 2017-07-21T17:32:28Z.
-            Only applicable for OpenShift-dedicated-metrics and OpenShift-metrics products"
-        - name: sort
-          in: query
-          schema:
-            $ref: "#/components/schemas/HostReportSort"
```

Comment:
Please remove this unused schema (HostReportSort) too.

Reply:
Thanks for spotting!